[material-ui][Button] Add loading feature to Button (#42987)
mui/material-ui: docs/data/material/components/button-group/button-group.md

{{"demo": "DisableElevation.js"}}

## Loading button

> You can use the `loading` prop to create loading buttons in button groups

Reviewer: ```suggestion
You can use the `loading` prop from `Button` to create loading buttons in button groups.
```

{{"demo": "DisableElevation.js"}}

> ## Loading button

Reviewer: ```suggestion
## Loading
```

> You can use the `loading` prop from `Button` to create loading buttons in button groups.

Reviewer: ```suggestion
Use the `loading` prop from `Button` to set buttons in a loading state and disable interactions.
```
